os/pm/pm_timedsuspend.c

```diff
@@ -108,9 +109,13 @@ static void timer_timeout(int argc, uint32_t domain, uint32_t pid)
  *
  ************************************************************************/
 
-int pm_timedsuspend(enum pm_domain_e domain, unsigned int timer_interval)
+int pm_timedsuspend(enum pm_domain_e domain, unsigned int milliseconds)
 {
 	int pid = PIDHASH(getpid());
+	int timer_interval;
+
```

Comment:
This value is not needed, and It has error in the pmvdbg

Please change as below

`	int ret = wd_start(wdog, MSEC2TICK(milliseconds), (wdentry_t)timer_timeout, 2, (uint32_t)domain, (uint32_t)pid);
	pmvdbg("PM is locked for pid %d and timer started for %d milisecond\n", pid, milliseconds);
`